src/calculateRentalCost.js

```diff
@@ -6,5 +1,22 @@
+const LONG_TERM = 3;
+const BASEPRISE = 40;
+const SELL = 20;
+
+const GOOD_DAYS = 7;
+const DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const fullCost = days * BASEPRISE;
+
+  if (days >= GOOD_DAYS) {
+    return fullCost - DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    return fullCost - SELL;
```

Comment:
The logic here applies a discount for rentals longer than `LONG_TERM`. Ensure that the variable `SELL` is correctly named to reflect its purpose as a discount.